### ISSUE TITLE
Requiring unknown module

### ISSUE THREAD

**Issue by tnguyen42** [facebook/react-native-fbsdk#11]
![image](https://cloud.githubusercontent.com/assets/12211685/9370304/8ef70356-46cf-11e5-928c-d96590b53e02.png)

Hi, I'm still having this weird problem where I can't use one third of the fbsdk:
- fbsdklogin doesn't work unless I got fbsdkcore installed
- fbsdkcore doesn't work unless I got fbsdkshare installed
- fbsdkshare doesn't work no matter what (cf screenshot)

It's not urgent at all but it would be nice if you could take a look at it.

**Comment by cphackm** [comment]
Version 0.0.4 of the SDK was just released today which changes how you include the packages in your Xcode project. There are xcodeproject files for each package now instead of bringing in the folders containing the source directly. I'd suggest updating(following the update setup instructions) if you're still having this issue.

**Comment by tnguyen42** [comment]
Thank you. I'll check this out and keep you updated.

**Comment by tnguyen42** [comment]
Hi... It's probably me missing something, but the compilation is failing and I get these errors:
![image](https://cloud.githubusercontent.com/assets/12211685/9430206/93d57b62-49ec-11e5-890d-b99a8b288b20.png)

I don't understand why the "file" (I don't know which one by the way, sorry for being a newbie) is trying to fetch something from `App/node_modules/react-native-facebook-login/FacebookSDK` when the module folder isn't even named react-native-facebook-login.
I assume I did something wrong somewhere... But I just don't get why. I just installed and followed the new procedure to install the sdk.
